pkg/csi_driver/reconciler.go

```diff
@@ -1240,8 +1231,17 @@ func (recon *MultishareReconciler) listMultishareResourceOps(ctx context.Context
 			continue
```

Comment:
if we fail to parse metadata we may risk skipping running ops?

Reply:
yeah but then we wouldn't be able to tell the target of the Op anyway: `file.IsInstanceTarget(meta.Target)`

Reply:
ok